.github/workflows/verify.yaml

```diff
@@ -166,5 +173,5 @@ jobs:
 
         run: |
           cd src
-          # docker compose pull verify
-          docker compose run verify
+          # Set GITHUB_WORKSPACE to set more log-friendly headers
```

Comment:
I am not sure what this envvar is supposed to do/how. If you want to use a var to detect the execution within GHA I would recommend using `GITHUB_ACTIONS` as this is a var that is set by GHA for this purpose vs `GITHUB_WORKSPACE` which usually contains the [workspace path](https://docs.github.com/en/actions/learn-github-actions/variables),